fs/fs_suite_test.go

```diff
@@ -12,7 +12,7 @@ import (
 
 func TestFs(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "fs Suite")
```

Comment:
What's the rationale for removing this word?

Reply:
Just getting it to sync up with the package names, when I run it on the command line.  It was getting kind of confusing when it said it was running `cmd Suite`, when that was really `main/cmd`.

```
$ ginkgo -r
[1524182441] main - 3/3 specs ••• SUCCESS! 136.296µs PASS
[1524182441] fs - 23/23 specs ••••••••••••••••••••••• SUCCESS! 8.187296ms PASS
[1524182441] http - 30/30 specs •••••••••••••••••••••••••••••• SUCCESS! 27.657094ms PASS
[1524182441] main/cmd - 21/23 specs •••••PP•••••••••••••••• SUCCESS! 314.743502ms PASS
```